projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/recommendations/row-types/ImageSizeRow.svelte

```diff
@@ -137,7 +138,16 @@
 			<p>{details.instructions}</p>
 			{#if details.page.edit_url}
 				<div class="jb-actions">
-					<Button width="auto" href={details.page.edit_url} fill>
+					<Button
+						width="auto"
+						fill
+						on:click={() =>
+							recordBoostEventAndRedirect(
+								details.page.edit_url,
+								'clicked_fix_on_page_on_isa_report',
```

Comment:
Thinking about this - maybe we should include the type of device as well? That way, we could later filter based on device type to see what's fixed most by users.